packages/mobile/src/pages/chat/edit-group-page.tsx

```diff
@@ -43,6 +43,18 @@ export function EditGroupPage(props: EditGroupProp) {
   const { mutate: introduceIdentities } = useIntroductions().introduceIdentities;
 
   const [newRecipients, setNewRecipients] = useState<string[]>([]);
+  const onRemove = useCallback(
+    (odinId: string) =>
+      setNewRecipients((currentRecipients) => currentRecipients.filter((x) => x !== odinId)),
+    []
+  );
+
+  const setContact = useCallback(
+    (newContact: string) => {
+      setNewRecipients([...newRecipients.filter((x) => x !== newContact), newContact]);
+    },
+    [newRecipients]
```

Comment:
You could use the same type of `setNewRecipients` like above, which avoids a new instance of setContact on every change of `newRecipients`